resources/lang/por.ini

```diff
@@ -3,22 +3,100 @@ language.name = "Portuguese"
+command.setrange.set = "Alcance da ferramenta foi definida para {%0}"
+command.biomeinfo.attarget = "Bioma no alvo"
+command.biomeinfo.atposition = "Bioma na posição"
+command.biomeinfo.result = "{%0} biomas encontrados na seleção"
+command.biomeinfo.result.line = "ID: {%0} Nome: {%1}"
+command.biomelist.title = "Lista de biomas"
+command.biomelist.result.line = "ID: {%0} Nome: {%1}"
+command.brushname.set = "Nome do pincel definido como \"{%0}\""
+command.clearclipboard.cleared = "Área de transferência limpa"
+command.flip.try = "Tentando virar a área de transferência em {%0}"
+command.flip.success = "Área de transferência lançada com sucesso"
+command.rotate.try = "Tentando girar a área de transferência em {%0} graus"
+command.rotate.success = "Área de transferência girada com sucesso"
+command.history.cleared = "Histórico limpo"
+command.listchunks.found = "{%0} pedaços encontrados na seleção"
+command.size = "Tamanho da selação"
+; selection
+selection.pos1.set = "Posição 1 difinido em X: {%0} Y: {%1} Z: {%2}"
+selection.pos2.set = "Posição 2 definido em X: {%0} Y: {%1} Z: {%2}"
+; session
+session.undo.none = "Nada para desfazer"
+session.undo.left = "Você tem {%0} ações para desfazer"
+session.redo.none = "Nada para refazer"
+session.redo.left = "Você tem  {%0} ações para refazer"
+session.brush.added = "Adicionado {%0} na sessão"
+session.brush.deleted = "Excluído {%0} (UUID {%1})"
+session.brush.removed = "Removido {%0} (UUID {%1})"
+session.language.set = "Definido com sucesso o idioma para {%0}"
+session.language.notfound = "Idioma {%0} não encontrado, redefinido para o padrão"
+; task
+task.copy.success = "Cópia assíncrona realizada com sucesso, levou {%0}, copiou {%1} blocos de um total de {%2}."
+task.count.success = "Análise assíncrona bem-sucedida, levou {%0}"
+task.count.result = "{%0} blocos encontrado em um total de {%1} blocos"
+task.fill.success = "Preenchimento assíncrono bem-sucedido, levou {%0}, {%1} blocos de um total de {%2} alterados."
+task.replace.success = "Substituição assíncrona bem-sucedida, levou {%0}, {%1} blocos de um total de {%2} alterados."
+task.revert.undo.success = "Desfeito assíncrono com êxito, levou {%0}, {%1} blocos de um total de {%2} alterados."
+task.revert.redo.success = "Refeito assíncrono com êxito, levou {%0}, {%1} blocos de um total de {%2} alterados."
+; flags
+flags.keepexistingblocks = "Manter blocos existentes"
+flags.keepair = "Manter ar"
+flags.hollow = "Oco"
+flags.hollowclosed = "Oco com extremidades fechadas"
+flags.natural = "Natural"
+; tools
+; wand tool
+tool.wand = "Varinha"
+tool.wand.lore.1 = "Clique com o botão esquerdo do mouse em um bloco para definir a posição 1 de uma seleção"
+tool.wand.lore.2 = "Clique com o botão direito do mouse em um bloco para definir a posição 2 de uma seleção"
+tool.wand.lore.3 = "Use //togglewand para alternar sua funcionalidade"
+tool.wand.disabled = "A ferramenta da varinha está desativada. Use //togglewand para reativá-lo"
```

Comment:
> Use

is still here

Reply:
Actually, it is correct. Use means use, in portuguese, but I can translate to other word, if you want.

Reply:
Intersting - one never stops learning i guess, thanks for clarifying